asteroid/dsp/beamforming.py

```diff
@@ -248,6 +248,91 @@ def compute_beamforming_vector(target_scm: torch.Tensor, noise_scm: torch.Tensor
         return bf_vect
 
 
+class GEVDBeamformer(Beamformer):
+    """Generalized eigenvalue decomposition speech distortion weighted multichannel Wiener filter.
+
+        Compare to SDW-MWF, spatial covariance matrix are computed from low rank approximation
+        based on eigen values decomposition,
+        see equation 62 in `[1] <https://hal.inria.fr/hal-01390918/file/14-1.pdf>`_.
+
+    Attributes:
+        mu (float): Speech distortion constant.
+        rank (int): Rank for the approximation of target covariance matrix,
+            no approximation is made if rank is None.
+
+    References:
+        [1] R. Serizel, M. Moonen, B. Van Dijk and J. Wouters,
+        "Low-rank Approximation Based Multichannel Wiener Filter Algorithms for
+        Noise Reduction with Application in Cochlear Implants,"
+        in IEEE/ACM Transactions on Audio, Speech, and Language Processing, April 2014.
+    """
+
+    def __init__(self, mu: float, rank: int):
```

Comment:
Make mu default to 1. 
How about rank? Default of 1, or None?

Reply:
For mu default should be 1.0, agree.
For the rank, 1 is the useful value but None make sense as well, I suggest to set to 1.

Reply:
1 it is then.